CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
-## 0.5.1-wip
+## 0.6.0-wip
```

Comment:
must it be a breaking change? That's...expensive

Reply:
There's two breaking changes here:

1. Removing the already broken `external` constructors. I believe even though they were broken, making them static errors constitutes a breaking change, but maybe I'm wrong.
2. Deprecating `createAudioElement`. This could potentially break Flutter if we don't roll the version number, although we could certainly defer the deprecation until later.

Reply:
The language is a bit vague here, but we may be able to get away without rolling a breaking change since these members were already broken: https://github.com/dart-lang/sdk/blob/main/docs/process/breaking-changes.md#dart-compatibility-and-philosophy.

> Implementation bugs: If the implementation deviates unintentionally from the specification, programs may break as we rectify the implementation.

Reply:
deprecations are no longer breaking for flutter. Or we could just put in an ignore.

Reply:
Ooh, can you link me to where that has changed? I've been following this: https://github.com/flutter/flutter/issues/108555 but there hasn't been any updates there. If that's the case, then we can just remove the version bump here, and if not, we can just defer the deprecation until that changes.

Reply:
@matanlurey ?

We could also just change the code we need to before we roll this.

Reply:
I don't have anything better to point to, but see https://github.com/flutter/flutter/issues/143312.

Reply:
Thanks Matan! Looks like the lint was added to the ignore list in all the relevant repos and Flutter doesn't use `createAudioElement` anywhere anyways, so I'll remove the version bump.